docs/getting-started.md

```diff
@@ -125,9 +125,16 @@ cd $Env:ProgramFiles\containerd\
 # - cni bin_dir and conf_dir locations
 Get-Content config.toml
 
+# Optional step to add the binaries (containerd.exe, ctr.exe) in $env:Path
+$Path = [Environment]::GetEnvironmentVariable("PATH", "Machine") + [IO.Path]::PathSeparator + $PWD.Path
```

Comment:
Minor nit: you could use `[System.EnvironmentVariableTarget]::Machine` instead of `"Machine"`, though I agree it is a bit less readable for virtually no gain, since there's zero chance for the `"Machine"` enum value to ever change...

Reply:
I like that, it's a little more concise, thanks!

Reply:
oops, not same behavour though, let me stick with the interface.
![image](https://user-images.githubusercontent.com/261265/225230560-00e0e1fc-189d-4a49-8407-25132c229222.png)